src/Providers/Abstractions/src/Embedding/IEmbeddingModel.cs

```diff
@@ -6,11 +6,6 @@ namespace LangChain.Providers;
 /// </summary>
 public interface IEmbeddingModel : IModel<EmbeddingSettings>
 {
-    /// <summary>
```

Comment:
Can you revert this? It's ok to use Settings in Amazon.Bedrock provider instead this, but not sure about other providers

Reply:
this is already removed in the latest commit